app/components/school-competencies-expanded.js

```diff
@@ -39,34 +39,32 @@ export default Component.extend({
     return competencyIds.length && ! isManaging;
   }),
 
-  didReceiveAttrs(){
+  async didReceiveAttrs(){
```

Comment:
We're overriding something on EmberObject here and I'm not sure it can be async. I guess javascript lets us, but nothing is going to resolve this promise so I'm not sure this is a good idea.

Reply:
i agree, but we're using promises already in this method already. switching to async/await syntax makes it more visible, imo. will throw a `@todo` comment in there. the whole thing will need to be re-done at some point, i'm afraid.

Reply:
Agreed.